packages/compass-components/src/components/guide-cue/guide-cue-service.ts

```diff
@@ -11,6 +11,11 @@ import {
 } from './guide-cue-storage';
 import { uniq } from 'lodash';
 
+// COMPASS-7357: jsdom `EventTarget` doesn't play nicely with Node.js `CustomEvent` yet,
+// so always pick the `window`/jsdom-based variant
+const EventTarget = globalThis.window?.EventTarget ?? globalThis.EventTarget;
+const CustomEvent = globalThis.window?.CustomEvent ?? globalThis.CustomEvent;
```

Comment:
Not a blocker, but we put everything that is only needed for jsdom in the configs/mocha-config-compass/register/jsdom-extra-mocks-register.js usually

Reply:
Sure, that makes sense to me – did this in 5e09df71fc09d1a506d58b2f7b1c001987b9786e, let's see what CI says 👍